app/src/main/java/com/framgia/englishconversation/screen/conversationdetail/ConversationDetailViewModel.java

```diff
@@ -57,17 +58,41 @@
     private ConversationDetailContract.Presenter mPresenter;
     private FragmentManager mManager;
     private CommentFragment mFragment;
+    private UserModel mTimelineUser;
+    private OnTimelineItemTouchListener mTouchListener =
+            new OnTimelineItemTouchListener<TimelineModel>() {
+                @Override
+                public void onItemTimelineClick(TimelineModel item) {
+                    //TODO
+                }
```

Comment:
ERROR: 'method def rcurly' have incorrect indentation level 16, expected level should be one of the following: 8, 12.